Merge project from sourceforge

axiopisty opened the pull request:
I converted the svn repository hosted on sourceforge to a git repository, and then merged that branch into master. The new github project already had a LICENSE file, which is the same license.txt file that existed in the sourceforge svn repository, so I removed the duplicate license.txt file.

Please note that I cannot currently build the project on my machine using `mvn clean install` in either the svn or the git repositories. This is because of the maven-apt-plugin, which is probably out dated on JVM 8.

So this pull request is merely to get the project from svn into git. I assume you have an environment in which the build will work successfully, and that you will know what changes need to occur to bring this project up to date on JVM 8.

I'm happy to help make those changes.

stoicflame committed:
initial import


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@1 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
adding a bunch of classes for the profile and serialization utilities.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@2 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
messages for the OFX profile
serialization logic
a few serialization tests.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@3 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
last checkin for support for the tagsoup reader.  now using nanoxml.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@4 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
deprecating the tagsoup parser.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@5 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
connections and initial services.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@6 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
adding statement download structure.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@7 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
moving around the domain classes.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@8 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
doing my best to test the profile against existing servers and stuff.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@9 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
other tests, reader enhancements.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@10 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
initial implementation of statement download.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@11 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
removing the line breaks.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@12 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
successfully downloading a statement.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@13 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
finalizing and testing transaction download.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@14 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
didn't need this directory.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@15 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
version 1.0


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@16 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
adding licence, notice, and copyright.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@17 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
version 1.1 release (adding account info request)


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@18 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
extracting an account details interface.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@19 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
adjustments for weird behavior in maven-apt-plugin checking for stale(!) sources.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@20 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
repo change to local deployment.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@21 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
checking in pom changes


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@22 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
removing the deprecation of TagSoup handlers.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@23 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
1.2


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@24 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
1.3 snapshot.


git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@27 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
investment support as submitted by Jon Perlow

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@28 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
update HEAD to 1.3, investment support as submitted by Jon Perlow

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@29 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
update HEAD to 1.4-SNAPSHOT

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@31 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
issue #3033085: closing tag for v1

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@32 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
#3033089: ofx statux exception

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@33 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
1.4

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@34 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
1.5 snapshot

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@36 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
1.5: additional patches

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@37 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
update head to snapshot

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@38 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
additional fixed for 1.5

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@39 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
update head to snapshot

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@40 4a7a7870-ee50-0410-826c-ac9ea6960489

Commit:
Finished implementation for the Profile Message Sets, including Banking, Billpay, CreditCard, Email, Interbank, Signup, WireTransfer.

Added Aggregate annotation to SignonInfo and generated sources to fix parsing error that blocked SignonInfoList from containing a list of SignonInfo objects.

Fixed Profile command line test tools to override validation, since we're passing null response objects. 

Added Bank of America profile test data and updated test case to test unmarshalling BoAs ProfileResponse message.

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@41 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
adding tax 1099 classes for RC1

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@42 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
RC2 changes

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@43 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
cutting 1.6

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@44 4a7a7870-ee50-0410-826c-ac9ea6960489

stoicflame committed:
updating head to 1.7 snapshot

git-svn-id: http://svn.code.sf.net/p/ofx4j/code/trunk@45 4a7a7870-ee50-0410-826c-ac9ea6960489

axiopisty committed:
Merge branch 'master' into svn-trunk

axiopisty committed:
Removed duplicate license file.

axiopisty committed:
Updated .gitignore

    - excluded intellij project files
    - excluded maven target directory

stoicflame commented:
Thanks for the patch. 

I've updated the project. I changed the package and group id from "net.sf.ofx4j" to "com.webcohesion.ofx4j" to make it easier to deploy (since I already had Sonatype's OSSRH permissions to use com.webcohesion.

I've also [created a README](https://github.com/stoicflame/ofx4j/blob/master/README.md) and updated the project at SourceForge to direct users to GitHub.

axiopisty commented:
That's great Ryan. Thank you. I was thinking it would be nice to configure
Travis CI and to move the documentation that is currently on SF to a GitHub
site for the project.

Anyway, I'll grab updates to see if it will build on my machine now. Once
it does I'll submit a patch for an OSGi bundle.

On Sep 14, 2016 5:40 PM, "Ryan Heaton" notifications@github.com wrote:

> Thanks for the patch.
> 
> I've updated the project. I changed the package and group id from
> "net.sf.ofx4j" to "com.webcohesion.ofx4j" to make it easier to deploy
> (since I already had Sonatype's OSSRH permissions to use com.webcohesion.
> 
> I've also created a README
> https://github.com/stoicflame/ofx4j/blob/master/README.md and updated
> the project at SourceForge to direct users to GitHub.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> https://github.com/stoicflame/ofx4j/pull/1#issuecomment-247190086, or mute
> the thread
> https://github.com/notifications/unsubscribe-auth/AC7o9jLd91wKzjea3IlMZ69YAJN3CM79ks5qqIXUgaJpZM4J69Qw
> .
